lib/inventory_refresh/save_collection/saver/base.rb

```diff
@@ -272,13 +272,18 @@ def assert_referential_integrity(hash)
       # @return [Time] A rails friendly time getting config from ActiveRecord::Base.default_timezone (can be :local
       #         or :utc)
       def time_now
-        if ActiveRecord::Base.default_timezone == :utc
+        if default_timezone_utc?
           Time.now.utc
         else
           Time.zone.now
         end
```

Comment:
Hm if the default_timezone is UTC then aren't `Time.now.utc` and `Time.zone.now` the same thing?  Could we simplify this to just `Time.zone.now`

Reply:
Oooh, I hear you saying, can we delete this?

```ruby
irb(main):009:0> Time.zone.now.class
=> ActiveSupport::TimeWithZone
irb(main):010:0> Time.now.utc.class
=> Time
```

Let me see if there is a reason it was written like this.

Reply:
It was localtime originally... maybe it's not needed at all.

❤️

Reply:
:scissors: :scissors: